plugins/afl-scoreboard/game_renderer.py

```
@@ -0,0 +1,550 @@
+        detail_config = customization.get('detail_text', {})
+        rank_config = customization.get('rank_text', {})
+        
+        try:
+            fonts["score"] = self._load_custom_font(score_config, default_size=10)
+            fonts["time"] = self._load_custom_font(period_config, default_size=8)
+            fonts["team"] = self._load_custom_font(team_config, default_size=8)
+            fonts["status"] = self._load_custom_font(status_config, default_size=6)
+            fonts["detail"] = self._load_custom_font(detail_config, default_size=6, default_font='4x6.ttf')
+            fonts["rank"] = self._load_custom_font(rank_config, default_size=10)
+            self.logger.debug("Successfully loaded fonts from config")
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)